docs/developer-docs-home.md

```diff
@@ -2,31 +2,61 @@
 
 ## Table of contents
 
-1. [Documentation index](#documentation-index)
+1. [What is a field plug-in](#what-is-a-field-plug-in)
+1. [What can a field plug-in do](#what-can-a-field-plug-in-do)
+1. [How does a field plug-in interact with a form](#how-does-a-field-plug-in-interact-with-a-form)
```

Comment:
@cking0987 , this item bullet says **How does a field plug-in...** but the actual section title is **How can a field plug-in...**. Which one is the correct one?

Reply:
Ah. Great catch. I think "How **can** a field..." is probably better.